manual_tests/test_5_full.yml

```diff
@@ -15,7 +15,7 @@
         env: dev
       trace.concentrator:
         extra_aggregators: version
-    datadog_checks:
+    agent_datadog_checks:
```

Comment:
for english would it be possible to have `datadog_agent` rather than `agent_datadog`?

Reply:
Can't do see above ^^